### PR TITLE
Update file for currency converting

### PR DESCRIPTION
**Pull request by snahornyi**
Related to https://github.com/prebid/Prebid.js/issues/6524
and
https://github.com/prebid/prebid-server/issues/1791

### CODE REVIEW

File: src/currencyRatesFileGen.js [hunk 1]
```diff
@@ -148,6 +146,16 @@ function pushToGithub(newDocument) {
     return (0);
 }
 
+function makeid(length) {
```
Comment (rpanchyk) [inline src/currencyRatesFileGen.js:149]: Please rename to `getRandomString`.

File: src/currencyRatesFileGen.js [hunk 2]
```diff
@@ -122,8 +118,10 @@ function pushToGithub(newDocument) {
     // change the cwd to /tmp
     process.chdir('/tmp')
     // clone the repository and set it as the cwd
-    runCommand(`git clone --quiet https://${gitRepositoryURL}`);
-    process.chdir(path.join(process.cwd(), repositoryName))
+    const randomPrefix = makeid(7);
```
Comment (rpanchyk) [inline src/currencyRatesFileGen.js:121]: Please add a comment why we need this random prefix.

File: src/currencyRatesFileGen.js [hunk 3]
```diff
@@ -262,21 +258,49 @@ function createDocumentParams(bucket, filename, documentObj, expires) {
...
+
+function createCurrencyResponse(currencyObject, currencyData, currencyDate) {
+    let response = Object.create({});
+    response["base"] = currencyObject.currency;
+    response["date"] = currencyDate;
+    let rate = Object.create({});
+    currencyData.forEach(data => {
+        rate[data.currency] = data.rate / currencyObject.rate;
+    })
+    rate["EUR"] = 1/ currencyObject.rate;
```
Comment (rpanchyk) [inline src/currencyRatesFileGen.js:297]: Pls add comment here why we need to add separate currency. Or even more, extract missing currencies to array and process them here. Up to you.

**Review by rpanchyk — CHANGES_REQUESTED** [review]

**Review by rpanchyk — APPROVED** [review]

**Comment by patmmccann** [comment]
This appears to publish the date it ran instead of the date in the ecb file. If ecb were to stop updating their file but continue publishing the xml, what check do we have?

**Review by patmmccann — CHANGES_REQUESTED** [review]
Can you change dataAsOf to pull the date from the source file?

**Comment by bretg** [comment]
@patmmccann - looks like your comment was incorporated - please re-review

**Comment by patmmccann** [comment]
Yes much appreciated, lgtm

**Pull request merged**